gateway-server/src/test/java/org/apache/knox/gateway/GatewayGlobalConfigTest.java

```diff
@@ -67,11 +69,20 @@ public void testSiteConfig() {
     GatewayConfig config = new GatewayConfigImpl();
     assertThat( config.getGatewayPort(), is( 5555 ) );
     assertThat( config.isClientAuthNeeded(), is( true ) );
+    assertTrue( config.isTopologyExcludedFromClientAuth("health"));
     assertThat( config.getTruststorePath(), is("./gateway-trust.jks"));
     assertThat( config.getTruststoreType(), is( "PKCS12" ) );
     assertThat( config.getKeystoreType(), is(KeyStore.getDefaultType()) );
   }
 
+  @Test
+  public void testSiteConfigWithDifferentTopologyExcluded() {
+    System.setProperty( GatewayConfigImpl.GATEWAY_HOME_VAR, getHomeDirName( "conf-site/conf/gateway-site.xml" ) );
+    GatewayConfig config = new GatewayConfigImpl();
+    assertThat( config.isClientAuthNeeded(), is( true ) );
```

Comment:
nit: `assertTrue` might have been better,  but doesn't prevent the merge this as-is